yacc.go

```diff
@@ -3331,18 +3323,17 @@ type $$Parser interface {
 }
 
 type $$ParserImpl struct {
-	lookahead func() int
+	lval  $$SymType
+	stack [16]$$SymType
```

Comment:
@tamird The array isn't reused. A new `ParserImpl` is allocated for every call to `Parse`.

@bdarnell Ok, I'll keep `stacksize` and rename to `initialStackSize`.